ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKStream.java

```diff
@@ -234,10 +235,14 @@ public KStream getKstream() {
 
   public String getExecutionPlan(String indent) {
     StringBuilder stringBuilder = new StringBuilder();
-    stringBuilder.append(indent + " > [ " + type + " ] Schema: " + SchemaUtil
-        .getSchemaDefinitionString(schema) + ".\n");
+    stringBuilder.append(indent)
+        .append(" > [ ")
+        .append(type).append(" ] Schema: ")
+        .append(SchemaUtil
+        .getSchemaDefinitionString(schema)).append(".\n");
```

Comment:
wouldn't it be better to make method invocation `SchemaUtil.getSchemaDefinitionString(schema)` and object definition on the same line?